lib/config/presets/internal/workarounds.ts

```diff
@@ -118,4 +119,21 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  javaLTSImageVersion: {
+    description: 'Limit Java runtime versions to LTS releases',
+    packageRules: [
+      {
+        matchDatasources: ['docker'],
+        matchPackageNames: ['openjdk', 'amazoncorretto'],
+        allowedVersions: '/^(11|17|21)(\\.[0-9]+){0,2}/',
```

Comment:
You're putting in the Java LTS versions : `(11|17|21)`. Those values will need to be updated manually later. 

Does the OpenJDK or Amazon Corretto team have a JSON or YAML file with their LTS schedule? For example: the Node.js project has a JSON file with their release schedule in it. Renovate then uses that schedule when offering upgrades.

- [`node-js-schedule.json` on our repository](https://github.com/renovatebot/renovate/blob/main/data/node-js-schedule.json)
- [GitHub Action workflow file that updates the static data](https://github.com/renovatebot/renovate/blob/main/.github/workflows/update-data.yml)

Just floating an idea, maybe there is no such schedule that we can use... 😄

Reply:
I'm not sure about Amazon Corretto, but Adoptium certainly does - https://api.adoptium.net/v3/info/available_releases - I've ween @viceice battling the stability of this API recently

Agree that using an API would certainly be a better solution for the original issue that @rarkins opened  (#8127)

This workaround would be temporary and is in response to this comment - https://github.com/renovatebot/renovate/issues/8127#issuecomment-1259651368

Reply:
LTS releases are every 2 years (or every 4 releases).  

I added the next LTS v21 in anticipation of it being released in Sep 2023.  Following this, v25 will be Sep 2025

Reply:
Yeah, this is a temporary and unfortunate workaround, but I think worth it

Reply:
If we can automate the "Java LTS schedule" update, or get an official source file with the schedule, we should do that. 😉 

The worst-case scenario is that we need to update one file every two years. We can probably live with that, in exchange for getting a new handy feature. 😄